specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -402,13 +416,19 @@ function Status({ mergingId }: { readonly mergingId: string }): JSX.Element {
   return (
     <Dialog
       buttons={
-        <Button.Danger onClick={undefined}>{wbText.stop()}</Button.Danger>
+        <Button.Danger onClick={() => setLoadingBar(false)}>
+          {wbText.stop()}
```

Comment:
@grantfitzsimmons should it say "Stop" or "Abort" (like in the WB if I remember correctly)

Reply:
we decided abort, I just used this one until we have an abort API